settings.gradle

```diff
@@ -20,3 +20,4 @@ rootProject.name = 'apache-spark-kubernetes-operator'
 include 'spark-operator-api'
 include 'spark-submission-worker'
 include 'spark-operator'
+include 'spark-operator-docs'
```

Comment:
Shall we move this into `build-tools` directory? In addition, `spark-operator-docs` sounds like a little overclaim because this has only `ConfOptionDocGenerator` while the documentations has more contents.

Reply:
Refactored to SPARK-49527